app/src/org/commcare/android/adapters/EntityDetailPagerAdapter.java

```diff
@@ -15,13 +15,14 @@
  * @author jschweers
  */
 public class EntityDetailPagerAdapter extends FragmentStatePagerAdapter {
-    
-    Detail detail;
-    int detailIndex;
-    boolean hasDetailCalloutListener;
-    TreeReference mEntityReference;
 
-    public EntityDetailPagerAdapter(FragmentManager fm, Detail detail, int detailIndex, TreeReference reference, boolean hasDetailCalloutListener) {    
+    private EntityDetailAdapter.EntityDetailViewModifier modifier;
+    final Detail detail;
+    final int detailIndex;
+    final boolean hasDetailCalloutListener;
```

Comment:
Isn't used (in the current branch, not sure about in the master restyling branch)